### DISCUSSION TITLE
Memory leak fixed in 1.5.9!

### DISCUSSION THREAD

**Discussion by silviumc** [SiaFoundation/siad#146] [General]
I've noticed something on my Cacti graphs: no more memory leak! It's there in the release notes: "Host: fixed a memory leak in RPC execute program". Great job fixing that! I had to make a script that was rebooting the machine when swap hit a limit. Screenshots attached. You can see when the update to 1.5.9 took place.
![memory](https://user-images.githubusercontent.com/106486855/181621345-15f13553-2096-4330-9b4d-a99b537f56dc.png)
![swap](https://user-images.githubusercontent.com/106486855/181621354-0e9f5cd2-8f7d-410c-9414-cc37b18664ce.png)

**Comment by lukechampine** [comment]
nice, thanks for sharing! It's always gratifying to see the concrete impact of fixes like this. :)
